clojure/defs.bzl

```diff
@@ -1,31 +1,45 @@
+# ; on windows, : on linux...
+_CLASSPATH_SEPARATOR = ";"
+
 def _impl(ctx):
     class_dir = ctx.actions.declare_directory(ctx.label.name + "/classes")
+    jar = ctx.actions.declare_file(ctx.label.name + ".jar")
+    args = ctx.actions.args()
+    args.add("-cp")
```

Comment:
What's the advantage of this builder over declaring the list like I did?

Reply:
Doing to_list on depsets of transitive deps is quadratic if done in each compile step.  The args thing is lazily materialized or something so that its more efficient.